.editorconfig

```diff
@@ -1,7 +1,7 @@
 root = true
 
 [*]
-indent_style = tab
+indent_style = space
```

Comment:
Looks like `scripts/format.bash` and `scripts/lint.bash` can't agree on how to use `.editorconfig`